devel/libfmt/Portfile

```diff
@@ -18,8 +18,8 @@ long_description    \
     fmt (formerly cppformat) is an open-source formatting library. \
     It can be used as a safe alternative to printf or as a fast alternative to C++ IOStreams.
 
-checksums           rmd160  cae7445443637ce7887bb6b2ef8136d7f57ded1b \
-                    sha256  4c322d3976cb4de91263fbcc1e8ddeffa6c5f32049b3ca2f740ba6a053df7052
+checksums           rmd160  7ab5001c8144d8d3d25b1726295e1447d065d694 \
+                    sha256  7b40e266c16cbcc16a9d8743713d012be3188872c4bce5dde3556cdf3b5846d2
```

Comment:
We now usually add a `size` parameter here. Other than that, I think your update is fine.

Reply:
Done!